src/main/xerus/monstercat/tabs/TabCatalog.kt

```diff
@@ -56,11 +61,70 @@ class TabCatalog : TableTab() {
 			it.next(); Settings.VISIBLECATALOGCOLUMNS.putMulti(*it.addedSubList.map { it.text }.toTypedArray())
 		})
 		table.setOnMouseClicked { me ->
-			if(me.clickCount == 2) {
-				val selected = table.selectionModel.selectedItem ?: return@setOnMouseClicked
-				Player.play(selected[cols.findUnsafe("Track")].trim(), selected[cols.findUnsafe("Artist")])
+			if(me.clickCount == 2 && me.button == MouseButton.PRIMARY) {
+				val selected = table.selectionModel.selectedItems ?: return@setOnMouseClicked
+				GlobalScope.launch {
+					Player.playTracks(getSongs(selected))
+				}
+			}else if(me.clickCount == 1 && me.button == MouseButton.MIDDLE){
```

Comment:
why is the clickCount relevant here?

Reply:
Can be usefull to prevent unwanted double-clicks I removed it in 42e81f91991a2d136268666a511726526b168b86